app/controllers/users_controller.rb

```diff
@@ -0,0 +1,36 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
+    if(@user.nil?)
+      flash[:danger] = t "error"
+    else
+      flash[:success] = t "welcome"
+    end
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+    if @user.save
+      flash[:success] = t "welcome"
+      redirect_to @user
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit :name,
+                                 :account_name,
+                                 :email,
+                                 :age,
+                                 :gender,
+                                 :password,
+                                 :password_confirmation
```

Comment:
```suggestion
    params.require(:user)
      .permit :name, :account_name, :email, :age, :gender, :password, :password_confirmation
```

Reply:
em sợ nó bị ruboco 80 character in a line á chị